db/articles.js

```diff
@@ -0,0 +1,16 @@
+const collection = [];
+
+function all() {
+  return collection;
+}
+
+function add(obj, productId) {
+  collection.push(obj);
+  collection[productId - 1].id = productId;
+  return collection;
+}
+
+module.exports = {
+  all: all,
```

Comment:
No real issue at all but just FYI, with ES6 if your key and value variable are the same you can do something like:
```
module.exports = {
  all,
  add
}
```